kitchen.yml

```diff
@@ -7,7 +7,7 @@ provisioner:
   data_bags_path: test/integration/data_bags
   encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
   enforce_idempotency: true
-  multiple_converge: 2
+  multiple_converge: 3
```

Comment:
Please change this only for the suite that's affected.

Reply:
Done